src/data/defaults.txt

```diff
@@ -1222,6 +1222,7 @@ user	schoolOfHardKnocksDiplomaCost	1000
 user	schoolSwagger	0
 user	scrapbookCharges	0
 user	screencappedMonster	
+user	scriptCascadingMenus	false
```

Comment:
I think defaulting this to true would be better -- I think it's more user friendly in two ways:
* showing all the scripts instead of nothing gives users another way to interact with scripts
* showing the cascading menu was the default before the bookmark change went in, so users don't have to change a setting to get back to the behaviour they were used to.

As there's no GUI for this at the moment (but perhaps there will be 😉), I think it's nicer for this to default to on. Your choice, though.

Reply:
User friendly depends upon which users you listen to :-)

Do we set the default for the existing users who are openly hostile to Bookmarks or do we set it for the new user who presumably should be encouraged to use Bookmarks?

How much does this default matter if we are likely to come up with something that lets a user select which one of the three options they want and easily switch between them?

I'm Ok with false.

Reply:
What the heck. I defaulted it to 2. It will shut up the openly hostile users, at least. :)